Phase1/FullTextSearch.java

```diff
@@ -2,29 +2,42 @@
 
 public class FullTextSearch {
     private String command;
+    private InvertedIndex invertedIndex;
 
     public FullTextSearch(String command) {
         this.command = command;
+        invertedIndex = new InvertedIndex();
     }
 
-    public void run() {
-        GetInput getInput = new GetInput("/home/melika/Desktop/codes/codestar/phase1/EnglishData");
-        HashMap<Integer, String> hashMap = getInput.readContent();
-        InvertedIndex invertedIndex = new InvertedIndex();
-
-        for (int id : hashMap.keySet()) {
-            String docString = hashMap.get(id);
-            Tokenizer tokenizer = new Tokenizer(docString);
-            HashSet<String> wordsSet = tokenizer.tokenize();
+    private void loadDocs() {
+        String FILE_ADDRESS = "Phase1/EnglishData";
```

Comment:
It's better to make this parameter to final because you write in the capital.

Reply:
What do you mean by final?

Reply:
What do you mean by final?

Reply:
looks like this:
**final** String **FILE_ADDRESS** = "Phase1/EnglishData";